petabtests/cases/v2.0.0/sbml/0011/0011.py

```diff
@@ -1,7 +1,7 @@
 from inspect import cleandoc
 
 import pandas as pd
-from petab.C import *
+from petab.v1.C import *
```

Comment:
Should this already import from `petab.v2`?

Reply:
For now it's the same. Once they diverge more, we'll have to re-organize test generation here. I'd wait until then.

Reply:
Well, I can also just update them now.

Reply:
When things break to due missing `petab.v2.C.PREEQUILIBRATION_CONDITION_ID`, we'll know we have to update the tests...

Reply:
Oops, there is no petab.v2.C yet